blocks/logo2/logo2.css

```diff
@@ -0,0 +1,12 @@
+.logo2 {
+  color: #283645;
```

Comment:
@ElenaPodv Could you please convert the color from hex to RGB? This will help maintain consistency with the chosen approach throughout our project.

Reply:
Done

Reply:
Done